v1/message.go

```diff
@@ -34,8 +34,9 @@ var (
 
 // Content contains the chat ID and the actual text of a message.
 type Content struct {
-	ChatID string `json:"chat_id"`
-	Text   string `json:"text"`
+	ChatID     string `json:"chat_id"`
+	Text       string `json:"text"`
+	ResponseTo string `json:"response-to"`
```

Comment:
Is it ok that `ChatID` uses snake case and `ResponseTo` uses dashes? I am fine with changing `chat_id` to `chat-id` if dashes are preferred.

Reply:
thanks, I think I will settle on a format once we have added all the fields, I have still to do a few changes on this.